oap-server/server-storage-plugin/storage-elasticsearch-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/elasticsearch/cache/NetworkAddressInventoryCacheEsDAO.java

```diff
@@ -84,7 +84,7 @@ public NetworkAddressInventoryCacheEsDAO(ElasticSearchClient client) {
         try {
             SearchSourceBuilder searchSourceBuilder = new SearchSourceBuilder();
             searchSourceBuilder.query(QueryBuilders.rangeQuery(NetworkAddressInventory.LAST_UPDATE_TIME).gte(lastUpdateTime));
-            searchSourceBuilder.size(500);
+            searchSourceBuilder.size(Integer.MAX_VALUE);
```

Comment:
Set source-builder's size to `Integer.MAX_VALUE ` may cause query time cost higher.

Reply:
@JaredTan95 Yes, for sure. But it is hard to guarantee to load all? If not, the update is not working.

Reply:
And every load has this condition
```java
List<NetworkAddressInventory> addressInventories = addressInventoryCacheDAO.loadLastUpdate(System.currentTimeMillis() - 60000);
```

The inventory is being reloaded only the update happens at the last minute. And timer runs every 10s(static config today)

Reply:
Yeah, I got your point.

Reply:
Considering our scenario here, the impact can be ignored in fact.